include/mbedtls/rsa.h

```diff
@@ -79,140 +84,142 @@ extern "C" {
 #endif
 
 /**
- * \brief   RSA context structure
+ * \brief   The RSA context structure.
  *
  * \note    Direct manipulation of the members of this structure
- *          is deprecated and will no longer be supported starting
- *          from the next major release. All manipulation should instead
- *          be done through the public interface functions.
- *
+ *          is deprecated. All manipulation should instead be done through 
+ *          the public interface functions.
  */
 typedef struct
 {
-    int ver;                    /*!<  always 0          */
-    size_t len;                 /*!<  size(N) in chars  */
+    int ver;                    /*!<  Always 0.*/
+    size_t len;                 /*!<  The size of \p N in characters. */
```

Comment:
Blocker: Please stick to `chars`, as the use of the C type name is deliberate here. Alternatively, we might use `Bytes`.

Reply:
Bytes would be more consistent with the rest of your documentation. Changed to Bytes